test_scripts/ATF_ShutDown_MASTER_RESET.lua

```diff
@@ -0,0 +1,148 @@
+--  Requirement summary:
+--  [Policies] Master Reset
+--
+--  Description:
+--  1. SDL and HMI are running
+--     App1 is registered
+--  2. Perform Master Reset
+--     HMI sends OnExitAllApplications with reason MASTER_RESET
+--
+--  Expected behavior:
+--  1. SDL clear all Apps folder, app_info.dat file and shut down
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local mobile_session = require('mobile_session')
+
+--[[ Local Variables ]]
+-- Hash id of AddCommand before MASTER_RESET
+local hash_id = nil
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/dummy_connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+
+function Test:StartSDL_With_One_Registered_App()
+  self:runSDL()
+  commonFunctions:waitForSDLStart(self):Do(function()
+    self:initHMI():Do(function()
+      commonFunctions:userPrint(35, "HMI initialized")
+      self:initHMI_onReady():Do(function ()
+        commonFunctions:userPrint(35, "HMI is ready")
+        self:connectMobile():Do(function ()
+          commonFunctions:userPrint(35, "Mobile Connected")
+          self:startSession():Do(function ()
+            commonFunctions:userPrint(35, "App is registered")
+          end)
+        end)
+      end)
+    end)
+  end)
+end
+
+function Test:Activate_App_And_Put_File()
+  local function addCommand()
+    local cid = self.mobileSession:SendRPC("AddCommand",{ cmdID = 1005, 
+                vrCommands = { "OnlyVRCommand"}
+                })
+    EXPECT_HMICALL("VR.AddCommand", {cmdID = 1005, type = "Command",
+                   vrCommands = {"OnlyVRCommand"}}):Do(function(_,data)
+      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+    EXPECT_RESPONSE(cid, { success = true, resultCode = "SUCCESS" })
+    EXPECT_NOTIFICATION("OnHashChange"):Do(function(_, data)
+      hash_id = data.payload.hashID
+    end)
+  end
+
+  commonSteps:ActivateAppInSpecificLevel(self, self.applications["Test Application"])
```

Comment:
@LitvinenkoIra , we should avoid using hard code "Test Application". It should be config.application1.registerAppInterfaceParams.appName

To avoid using very long name, we can define s consts.lua file and use short variable name
`default_app_name = config.application1.registerAppInterfaceParams.appName
`